intro_flutter_gpu/step_09/lib/main.dart

```diff
@@ -198,4 +148,42 @@ class TrianglePainter extends CustomPainter {
 
   @override
   bool shouldRepaint(covariant CustomPainter oldDelegate) => true;
+
+  static const List<List<double>> vertices = [
+    // Vertex format: [x, y, z, u, v, w]
+
+    // Front face of cube
+    [-0.5, -0.5, 0.5, 0.0, 0.0, 1.0],
+    [0.5, -0.5, 0.5, 1.0, 0.0, 1.0],
+    [0.5, 0.5, 0.5, 1.0, 1.0, 1.0],
+    [-0.5, 0.5, 0.5, 0.0, 1.0, 1.0],
+
+    // back face of cube
+    [-0.5, -0.5, -0.5, 0.0, 0.0, 0.0],
+    [0.5, -0.5, -0.5, 1.0, 0.0, 0.0],
+    [0.5, 0.5, -0.5, 1.0, 1.0, 0.0],
+    [-0.5, 0.5, -0.5, 0.0, 1.0, 0.0],
+  ];
+
+  // Define indices for triangles (counter-clockwise winding)
+  static const indices = [
```

Comment:
To be consistent with the other static fields you added:

```suggestion
  static const List<int> indices = [
```